### PR TITLE
ROX-15287: fix stackrox master branch ocp 4.6 testing

### PR DESCRIPTION
**Pull request by BradLugo**
/cc @gavin-stackrox

### CODE REVIEW

**Comment by BradLugo** [comment]
/pj-rehearse pull-ci-stackrox-stackrox-master-ocp-4-6-operator-e2e-tests pull-ci-stackrox-stackrox-master-ocp-4-6-qa-e2e-tests pull-ci-stackrox-stackrox-master-ocp-4-6-ui-e2e-tests

File: ci-operator/config/stackrox/stackrox/stackrox-stackrox-master__ocp-4-6.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+      version: "4.6"
+resources:
+  '*':
+    requests:
+      cpu: 2000m
+      memory: 4000Mi
+test_binary_build_commands: .openshift-ci/dispatch.sh test-binary-build-commands
+tests:
+- always_run: false
+  as: qa-e2e-tests
```
Comment (gavin-stackrox) [inline ci-operator/config/stackrox/stackrox/stackrox-stackrox-master__ocp-4-6.yaml:24]: Nice! gets rid of that echo :)

File: ci-operator/config/stackrox/stackrox/stackrox-stackrox-master__ocp-4-6.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  optional: true
+  run_if_changed: ^((generated|image|operator|pkg|make|deploy|scripts)/.*|Makefile|status\.sh|EXPECTED_GO_VERSION)
```
Comment (gavin-stackrox) [inline ci-operator/config/stackrox/stackrox/stackrox-stackrox-master__ocp-4-6.yaml:26]: 4.6 maser presubmit is ! run_if_changed per https://docs.engineering.redhat.com/display/StackRox/ACS+-+OpenShift+version+support
```suggestion
```

File: ci-operator/config/stackrox/stackrox/stackrox-stackrox-master__ocp-4-6.yaml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    env:
+      TEST_SUITE: openshift-newest-qa-e2e-tests
+    test:
+    - ref: stackrox-e2e
+    workflow: ipi-aws
+  timeout: 5h0m0s
+- always_run: false
+  as: operator-e2e-tests
+  optional: true
+  run_if_changed: ^((generated|image|operator|pkg|make|deploy|scripts)/.*|Makefile|status\.sh|EXPECTED_GO_VERSION)
```
Comment (gavin-stackrox) [inline ci-operator/config/stackrox/stackrox/stackrox-stackrox-master__ocp-4-6.yaml:38]: ditto
```suggestion
```

File: ci-operator/jobs/stackrox/stackrox/stackrox-stackrox-master-presubmits.yaml [hunk 4]
```diff
@@ -1225,27 +1225,33 @@ presubmits:
...
+      ci-operator.openshift.io/cloud: aws
+      ci-operator.openshift.io/cloud-cluster-profile: aws
+      ci-operator.openshift.io/variant: ocp-4-6
       ci.openshift.io/generator: prowgen
-      job-release: "4.13"
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
     name: pull-ci-stackrox-stackrox-master-ocp-4-6-operator-e2e-tests
     optional: true
     rerun_command: /test ocp-4-6-operator-e2e-tests
+    run_if_changed: ^((generated|image|operator|pkg|make|deploy|scripts)/.*|Makefile|status\.sh|EXPECTED_GO_VERSION)
```
Comment (gavin-stackrox) [inline ci-operator/jobs/stackrox/stackrox/stackrox-stackrox-master-presubmits.yaml:1243]: you may need to manually delete this after removing in config and running `make update`.

**Review by gavin-stackrox — CHANGES_REQUESTED** [review]

**Review by gavin-stackrox — APPROVED** [review]
/lgtm

**Comment by gavin-stackrox** [comment]
/cc @msugakov 
FYI

**Pull request merged**